oauth2/oauth2-resource-server/src/main/java/org/springframework/security/oauth2/server/resource/authentication/JwtAuthenticationToken.java

```diff
@@ -56,6 +70,19 @@ public JwtAuthenticationToken(Jwt jwt, Collection<? extends GrantedAuthority> au
 		this.setAuthenticated(true);
 	}
 
+	/**
```

Comment:
Why did `(Jwt, Collection)` not get changed? Seems like we'd need to set the name there, too.